### ISSUE TITLE
/README.md's APE section

### ISSUE THREAD

**Issue by gherson** [AceWiki/AceWiki#24]
Regarding /README.md's APE section:

- It says 'the file "ape.exe" has to be available'.  Should it also mention that this file belongs in AceWiki's top folder?
- Is it necessary that "The jpl.jar file is no longer part of the AceWiki package"?
- Shouldn't it be mentioned that APE-master\java\target\attempto-ape-_version_.jar must be copied to AceWiki/lib, and `;lib/attempto-ape-version.jar` must be added to the `javac` element's `classpath` attribute in AceWiki/build.xml?  Or is that not necessarily the case?

**Comment by gherson** [comment]
Maybe also mention that Java >= 1.7 is required since
AceWiki\src\ch\uzh\ifi\attempto\acewiki\Wiki.java line 792 has the diamond operator introduced with Java 1.7 per Issue #27:
List<WikiPage> pages = new ArrayList<>();

**Comment by tkuhn** [comment]
> It says 'the file "ape.exe" has to be available'. Should it also mention that this file belongs in AceWiki's top folder?

Right. I added that now (698405e993500d275106e2396f0caf55d58802d6).

> Is it necessary that "The jpl.jar file is no longer part of the AceWiki package"?

Yes, it has to be the same version as the `swipl` command.

> Shouldn't it be mentioned that APE-master\java\target\attempto-ape-version.jar must be copied to AceWiki/lib, and ;lib/attempto-ape-version.jar must be added to the javac element's classpath attribute in AceWiki/build.xml? Or is that not necessarily the case?

The ape jar file is already part of the AceWiki lib directory: https://github.com/AceWiki/AceWiki/blob/master/lib/attempto-ape.jar

It shouldn't be necessary to copy over the latest version. It hasn't changed much, and if so only in a backwards compatible manner. Or did you experience problems when not using the latest version of the ape jar?

> Maybe also mention that Java >= 1.7 is required

Yes. Done!
